gensim/test/test_corpora.py

```diff
@@ -215,10 +215,10 @@ def _get_slice(corpus, slice_):
             self.assertRaises(RuntimeError, _get_slice, corpus_, 1.0)
 
 
-class TestMmCorpus(CorpusTestCase):
+class TestMmCorpusWithIndex(CorpusTestCase):
```

Comment:
what's about backward compatibility tests?

Reply:
I made sure all the tests pass with the existing version of MmReader as well, so there shouldn't be any backward compatibility issues.